src/v1/optimizer/logical_plan_rewriter/arith_expr_simplification.rs

```diff
@@ -105,38 +105,3 @@ impl PlanRewriter for ArithExprSimplificationRule {
         Arc::new(plan.clone_with_rewrite_expr(self))
     }
 }
-
-#[cfg(test)]
-mod tests {
-    use std::fs;
-
-    use manifest_dir_macros;
-    use serde::Serialize;
-
-    use crate::Database;
-
-    #[tokio::main]
-    #[test]
-    async fn test_expr_simplification() {
```

Comment:
Why is it removed?

Reply:
I thought it had been replaced by sqlplannertest?